front/pages/w/[wId]/assistant/labs/transcripts/index.tsx

```diff
@@ -84,32 +90,39 @@ export default function LabsTranscriptsIndex({
     transcriptsConfiguration,
     isTranscriptsConfigurationLoading,
     mutateTranscriptsConfiguration,
-  } = useLabsTranscriptsConfiguration({
-    workspaceId: owner.sId,
-    provider,
-  });
+  } = useLabsTranscriptsConfiguration({ workspaceId: owner.sId });
 
   const [transcriptsConfigurationState, setTranscriptsConfigurationState] =
     useState<{
+      provider: string;
       isGDriveConnected: boolean;
+      isGongConnected: boolean;
       assistantSelected: LightAgentConfigurationType | null;
       isActive: boolean;
     }>({
+      provider: transcriptsConfiguration?.provider || "",
       isGDriveConnected: false,
-      assistantSelected: null as LightAgentConfigurationType | null,
+      isGongConnected: false,
+      assistantSelected: null,
       isActive: false,
     });
 
   useEffect(() => {
-    setTranscriptsConfigurationState({
-      isGDriveConnected:
-        (transcriptsConfiguration && transcriptsConfiguration.id > 0) || false,
-      assistantSelected:
-        agentConfigurations.find(
-          (a) => a.sId === transcriptsConfiguration?.agentConfigurationId
-        ) || null,
-      isActive: transcriptsConfiguration?.isActive || false,
-    });
+    if (transcriptsConfiguration) {
+      setTranscriptsConfigurationState({
+        ...transcriptsConfigurationState,
+        provider: transcriptsConfiguration.provider || "",
+        isGongConnected: transcriptsConfiguration.provider == "gong" || false,
+        isGDriveConnected:
+          transcriptsConfiguration.provider == "google_drive" || false,
+        assistantSelected:
+          agentConfigurations.find(
+            (a) => a.sId === transcriptsConfiguration.agentConfigurationId
+          ) || null,
+        isActive: transcriptsConfiguration.isActive || false,
+      });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why do we need to disable exhaustive deps?

Reply:
Per IRL, we can now remove the comment since use the proper way of updating the state, which makes us NOT depend on `transcriptsConfigurationState`